web/js/codeworld_shared.js

```diff
@@ -532,13 +532,36 @@ function registerStandardHints(successFunc) {
       found[2] = [];
     }
 
-    const options = found[0].concat(found[1]).concat(found[2]);
+    const mappedTerms = {
+      square: 'rectangle',
```

Comment:
It would be slightly more correct to map these to `Prelude.rectangle` and `Prelude.polygon`.  That's still not 100% right, since someone could presumably do:

    import Prelude ()
    import qualified SomeOtherModule as Prelude

but we can assume that anyone doing that is trying to fool the system, so that's fine.  But I can definitely see someone in, for example, LSU's class that uses their own standard library running into problems with this mapping.